### ISSUE TITLE
md-input-container render fix for only right icon

### ISSUE THREAD

**Issue by Phr33d0m** [angular/material#6896]
Hi, if I want to add an icon only on the right side of the input it doesn't render well:

![render-off](https://cloud.githubusercontent.com/assets/1591486/12648517/aaef6cc6-c5da-11e5-8d8e-b6eb991eb863.png)

Example structure:
```html
<md-input-container class="md-icon-right">
	<label>Price</label>
	<input type="text" ng-model="price"></input>
	<md-icon md-font-icon="mdi mdi-currency-eur"></md-icon>
</md-input-container>
```

Proposed CSS fix:
```css
.md-has-icon {
	&.md-icon-right {
		padding-left: 0;

		& > label {
			left: 0;
		}
	}
}
```

Renders correctly:

![render-ok](https://cloud.githubusercontent.com/assets/1591486/12648541/c6d20bf6-c5da-11e5-90a7-e4a64477adde.png)

Regards.

**Comment by DevVersion** [comment]
This seems to be fixed in #6720 (@EladBezalel) - tested it, works perfect in #6720

**Comment by ThomasBurleson** [comment]
Fixed with SHA 0c87f08
